packages/typedefs/src/interaction.ts

```diff
@@ -0,0 +1,85 @@
+
+export const UNKNOWN /*                       */ = "UNKNOWN"
+export const SCRIPT /*                        */ = "SCRIPT"
+export const TRANSACTION /*                   */ = "TRANSACTION"
+export const GET_TRANSACTION_STATUS /*        */ = "GET_TRANSACTION_STATUS"
+export const GET_ACCOUNT /*                   */ = "GET_ACCOUNT"
+export const GET_EVENTS /*                    */ = "GET_EVENTS"
+export const PING /*                          */ = "PING"
+export const GET_TRANSACTION /*               */ = "GET_TRANSACTION"
+export const GET_BLOCK /*                     */ = "GET_BLOCK"
+export const GET_BLOCK_HEADER /*              */ = "GET_BLOCK_HEADER"
+export const GET_COLLECTION /*                */ = "GET_COLLECTION"
+export const GET_NETWORK_PARAMETERS /*        */ = "GET_NETWORK_PARAMETERS"
+
+export const BAD /* */ = "BAD"
+export const OK /*  */ = "OK"
+
+export const ACCOUNT /*  */ = "ACCOUNT"
+export const PARAM /*    */ = "PARAM"
+export const ARGUMENT /* */ = "ARGUMENT"
+
+export const AUTHORIZER /* */ = "authorizer"
+export const PAYER /*      */ = "payer"
+export const PROPOSER /*   */ = "proposer"
+
+export interface IAcct {
+  "kind": typeof ACCOUNT,
+  "tempId": string | null,
+  "addr": string | null,
+  "keyId": number | string | null,
+  "sequenceNum": number | null,
+  "signature": any | null,
+  "signingFunction": any | null,
+  "resolve": any | null,
+  "role": {
+    "proposer": boolean,
+    "authorizer": boolean,
+    "payer": boolean,
+    "param"?: boolean,
+  },
+  authorization: any,
+}
+
+export interface IIx {
```

Comment:
nit: can we use a clearer interface name?  I.e. `Interaction`?  `SdkInteraction`?